examples/inprocess_qtconsole.py

```diff
@@ -10,7 +10,7 @@
 """
 
 
-from qtconsole.qt import QtGui
+from qtpy import QtGui
```

Comment:
This needs to be

    from qtpy import QtWidgets

to use the right import below.